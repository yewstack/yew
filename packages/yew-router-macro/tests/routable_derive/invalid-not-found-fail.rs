#[derive(Debug, PartialEq, yew_router::Routable)]
enum RoutesOne {
    #[at("/")]
    #[not_found]
    Home,
    #[at("/404")]
    #[not_found]
    NotFound,
}

#[derive(Debug, PartialEq, yew_router::Routable)]
enum RoutesTwo {
    #[at("/404")]
    #[not_found]
    #[not_found]
    NotFound,
}
fn main() {}
