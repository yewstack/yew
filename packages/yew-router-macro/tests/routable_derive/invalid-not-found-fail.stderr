error: there can only be one not_found
 --> $DIR/invalid-not-found-fail.rs:4:5
  |
4 | /     #[not_found]
5 | |     Home,
6 | |     #[at("/404")]
7 | |     #[not_found]
  | |________________^

error: there can only be one not_found
  --> $DIR/invalid-not-found-fail.rs:14:5
   |
14 | /     #[not_found]
15 | |     #[not_found]
   | |________________^
